obs-server.sls

```diff
@@ -0,0 +1,65 @@
+install obs server packages:
+  pkg.installed:
+    - pkgs:
+      - obs-server
+      - obs-worker
+
+obs-api:
+  pkg:
+    - installed
+  file.managed:
+     - name: /usr/share/obs/api/config/database.yml
+     - source: salt://llvm-obs/obs-conf/obs-api_database.yml
+
+populate_database setup db:
+  cmd.run:
+     - name: "RAILS_ENV=\"production\" rake db:setup"
+     - cwd: /usr/share/obs/api/
+
+populate_database write configuration:
+  cmd.run:
+     - name: "RAILS_ENV=\"production\" rake writeconfiguration"
+     - cwd: /usr/share/obs/api/
+
+change permission for log:
```

Comment:
I think you can do that with salt without using cmd.run?

Reply:
I could not find much on this. May you please suggest something ?

Reply:
```
/usr/share/obs/api/log:
  file.directory:
    - user: www-data
    - group: www-data

```
Doesn't work?

Reply:
I was confused with "RAILS_ENV=\"production\" rake writeconfiguration", updated the same. Thanks :)